app/views/spam2/_comments.html.erb

```diff
@@ -0,0 +1,74 @@
+<script>
+$(document).ready(function() {
```

Comment:
This block needs to be formatted and documented a bit.

Reply:
I formatted it and added some comments